### PR TITLE
Mobile projects/about

### PR DESCRIPTION
**Pull request by cuauh-cabrera**
### Below are listed all the features included in this PR:

- [ ] Add a projects section for the mobile version.
- [ ] Add an about me section for the mobile version.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -105,3 +108,231 @@ ol a {
...
+  margin-top: 24px;
+}
+
+.projects-container {
+  width: 95%;
+  margin-top: 38px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```
Comment (amon-cofie) [inline styles.css]: - [x] Great use of semantic HTML 👍. Setting the display to flex works just fine here but you would be surprised how easy a display of grid helps when you have to remodel the layout of a bunch of cards depending on the screen size. It makes repositioning so much easier and helps make your code clean and DRY when used properly. I suggest that you use a display of grid here instead 😉. You can find out more about display grid [here](https://www.w3schools.com/css/css_grid.asp)

File: styles.css [hunk 2]
```diff
@@ -105,3 +108,231 @@ ol a {
...
+  height: 48px;
+  background-color: rgba(54, 179, 127, 1);
+  text-align: center;
+  font-size: 1em;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: rgba(255, 255, 255, 1);
+}
```
Comment (amon-cofie) [inline styles.css:222]: - [x] Your buttons look very well styled 👍. However, I suggest that you add the styling for the active, hover and pressed states of the buttons as indicated in the figma design. I will add a screenshot below: 

image

File: index.html [hunk 3]
```diff
@@ -51,6 +51,171 @@ <h1 class="headline__title"><span class="headline__title--black">Hey there. <br>
...
+                        <ul>
+                            <li>Database Management</li>
+                            <li>Version Control</li>
+                            <li>CLI</li>
+                            <li>Web Development</li>
+                            <li>API Design</li>
+                        </ul>
+                    </div>
+                </article>
+            </div>
```
Comment (amon-cofie) [inline index.html]: - [ ] Well done! It is evident that you have a very high interest in semantics and clean code 👍. However, I believe that these three articles are each a technology category or I should say these articles are a ```list``` of technology categories 😉. In that light, I suggest that you put these articles in a list ie. ul tags make it more semantically sound 👍. Your code might end up looking something like this: 

```sh
<ul class="about__card-container">
                <!--Languages Card-->
                <li>
                <article class="about__card">
                    <div class="about__card--icon">
                        <img src="./assets/languages_icon.png" alt="">
                    </div>
                    <div class="about__card--title">
                        <h3>Languages</h3>
                    </div>
                    <div class="about__card--tags">
                        <ul>
                            <li>JavaScript</li>
                            <li>Ruby</li>
                            <li>HTML</li>
                            <li>CSS</li>
                        </ul>
                    </div>
                </article>
                </li>
                <!--Frameworks Card-->
                <li>
                <article class="about__card">
                    <div class="about__card--icon">
                        <img src="./assets/icon-frameworks.png" alt="">
                    </div>
                    <div class="about__card--title">
                        <h3>Frameworks</h3>
                    </div>
                    <div class="about__card--tags">
                        <ul>
                            <li>React.js</li>
                            <li>Ruby on Rails</li>
                            <li>RSpec</li>
                            <li>Capybara</li>
                            <li>Selenium</li>
                        </ul>
                    </div>
                </article>
                </li>
                <!--Skills Card-->
                <li>
                <article class="about__card">
                    <div class="about__card--icon">
                        <img src="./assets/icon-skills.png" alt="">
                    </div>
                    <div class="about__card--title">
                        <h3>Skills</h3>
                    </div>
                    <div class="about__card--tags">
                        <ul>
                            <li>Database Management</li>
                            <li>Version Control</li>
                            <li>CLI</li>
                            <li>Web Development</li>
                            <li>API Design</li>
                        </ul>
                    </div>
                </article>
                </li>
</ul>
```

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi @cuauh-cabrera , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! 💪 



## Highlights

🚀 You organized your code very well 
🚀 You made excellent use of semantic HTML
🚀 You have good commit messages
🚀 You used CSS classes very well and kept your code DRY
🚀 You avoided the use of CSS id selectors for styling 👍 
🚀 You have no linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @cuauh-cabrera, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by LionRouge1 — APPROVED** [review]

**Pull request merged**